src/cow-react/modules/swap/containers/EthFlowStepper/index.tsx

```diff
@@ -31,18 +32,18 @@ export function EthFlowStepper(props: EthFlowStepperProps) {
       createOrderTx: order.orderCreationHash || '',
       orderId: order.id,
       state,
-      isExpired: order.status === 'expired',
+      isExpired: isEthFlowOrderExpired(order),
       // rejectedReason?: TODO: address when dealing with rejections
     },
     // TODO: fill these in when dealing with rejections
     refund: {
       // refundTx?: string
       isRefunded: false,
     },
-    // TODO: fill these in when dealing with cancellations
-    cancelation: {
-      // cancelationTx?: string
-      isCanceled: false,
+    cancellation: {
+      cancellationTx: order.cancellationHash,
+      //  TODO: wire this up also with the cancellation tx once that's implemented
```

Comment:
Next PR.
Right now state is only updated if order exists on backend and is cancelled there.